python/pyngraph/ops/distbroadcast.hpp

```diff
@@ -0,0 +1,23 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+void regclass_pyngraph_op_DistBroadcast(py::module m);
```

Comment:
Needs to be called in python/pyngraph/ops/regmodule_pyngraph_op.cpp

Reply:
Thanks! Added the op call in `python/pyngraph/ops/regmodule_pyngraph_op.cpp`.